draft/overlay/raspihats-i2c-hat-di6acdq6rly.md

```diff
@@ -0,0 +1,58 @@
+<!--
+---
+name: DI6acDQ6rly I2C-HAT
+class: board
+type: io, relay
+formfactor: Custom
+manufacturer: Raspihats
+description: 6 isolated digital input channels and 6 relay output channels add-on board for the Raspberry Pi 
+url: http://raspihats.com/product/di6acdq6rly-i2c-hat
+buy: http://raspihats.com/product/di6acdq6rly-i2c-hat
+image: 'raspihats-di6acdq6rly-i2c-hat.png'
```

Comment:
Fixed, but I'm wondering, maybe it's better to change all file names to _raspihats-di6acdq6rly-i2c-hat.*_ ?